docs/src/modules/ROOT/pages/quickstart/hello-world/hello-world-quickstart.adoc

```diff
@@ -391,19 +395,326 @@ public class TimetableApp {
 
 }
 ----
+--
+
+Kotlin::
++
+--
+Create the `src/main/kotlin/org/acme/schooltimetabling/TimetableApp.kt` class:
+
+[source,kotlin]
+----
+package org.acme.schooltimetabling
+
+import ai.timefold.solver.core.api.solver.SolverFactory
+import ai.timefold.solver.core.config.solver.SolverConfig
+import org.acme.schooltimetabling.domain.Lesson
+import org.acme.schooltimetabling.domain.Room
+import org.acme.schooltimetabling.domain.Timeslot
+import org.acme.schooltimetabling.domain.Timetable
+import org.acme.schooltimetabling.solver.TimetableConstraintProvider
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import java.time.DayOfWeek
+import java.time.Duration
+import java.time.LocalTime
+import java.util.Objects
+import java.util.function.Function
+import java.util.stream.Collectors
+
+object TimetableApp {
+    private val LOGGER: Logger = LoggerFactory.getLogger(TimetableApp::class.java)
+
+    @JvmStatic
+    fun main(args: Array<String>) {
+        val solverFactory = SolverFactory.create<Timetable>(
+            SolverConfig()
+                .withSolutionClass(Timetable::class.java)
+                .withEntityClasses(Lesson::class.java)
+                .withConstraintProviderClass(TimetableConstraintProvider::class.java)
+                // The solver runs only for 5 seconds on this small dataset.
+                // It's recommended to run for at least 5 minutes ("5m") otherwise.
+                .withTerminationSpentLimit(Duration.ofSeconds(5))
+        )
+
+        // Load the problem
+        val problem = generateDemoData(DemoData.SMALL)
+
+        // Solve the problem
+        val solver = solverFactory.buildSolver()
+        val solution = solver.solve(problem)
+
+        // Visualize the solution
+        printTimetable(solution)
+    }
+
+    fun generateDemoData(demoData: DemoData): Timetable {
+        val timeslots: MutableList<Timeslot> = ArrayList(10)
+        var nextTimeslotId = 0L
+        timeslots.add(Timeslot(nextTimeslotId++, DayOfWeek.MONDAY, LocalTime.of(8, 30), LocalTime.of(9, 30)))
```

Comment:
There is no `id` property for `Timeslot`. We should either update the model class or remove `nextTimeslotId++`.